### ISSUE TITLE
Random pop up

### ISSUE THREAD

**Issue by twit4444** [New-Chance-RP/Feedback#723]
<h2>What bug have you found?</h2>

In the mount Zona garage i fou8nd something that doesn't seem like it should be there, here is clip https://youtu.be/1fXA--CZW5Y

You don't have to fix it right away because it honestly is not that big of a deal but just something that i thought seemed weird

**Comment by greed702** [comment]
That's the marker for a vehicle. Say you can pull out a vehicle and someone else pulls one out as well, they won't land on top of each other so you can pull out multiple vehicles at once and not have to wait. Also doubles as a "don't park here" reminder.

**Comment by Mubbb** [comment]
I believe they're just pre determined spawn locations for garage vehicles as well as probably working as spawn logic markers so cars dont spawn it theres a car on the market etc. Wouldnt really say it needs changing tbh but that's just me

**Comment by Senlarus** [comment]
Correct, I will go into the script and remove the notification popups for the additional parking spots that were defined so you'll only get it near the storage circle.
